.github/workflows/test-build-binaries.yml

```
@@ -27,67 +27,36 @@
-
-      - name: Install poetry
-        run: |
-          curl -sSL https://install.python-poetry.org | python - -y
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
+        with:
+          enable-cache: true
+          cache-dependency-glob: "uv.lock"
 
       - name: Update Path for Windows
         run: echo "$APPDATA\Python\Scripts" >> $GITHUB_PATH
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:35: Double quote to prevent globbing and word splitting

(shellcheck)